addons/account_edi/models/account_move.py

```diff
@@ -17,7 +17,7 @@ class AccountMove(models.Model):
         inverse_name='move_id')
     edi_state = fields.Selection(
         selection=[('to_send', 'To Send'), ('sent', 'Sent'), ('to_cancel', 'To Cancel'), ('cancelled', 'Cancelled')],
-        string="Electronic invoicing",
+        string="Edi State",
```

Comment:
EDI State or Electronic Invoicing State

Reply:
@aliyatastemirova I told you @jco-odoo would have said that.

First, this morning we had a case of someone not aware of the "you won't be able to change the demo flag" warning because it was in English only.

Second, read what Aliya says about it.
![image](https://user-images.githubusercontent.com/1665365/233113038-efb00e4a-e610-477f-81f6-18100a3602a1.png)

Reply:
Translation is an ABSOLUTE priority, trust me.

Reply:
Maybe the next guy who will touch `account_edi` will then just have to translate what he does, instead of demanding that to future developers or Transifex.